src/build_targets.rs

```diff
@@ -164,13 +179,19 @@ impl FileNames {
 
                 if target.env == "msvc" {
                     let static_lib = targetdir.join(format!("{lib_name}.lib"));
-                    let impl_lib = targetdir.join(format!("{lib_name}.dll.lib"));
+                    let impl_lib = match use_meson_naming_convention {
+                        false => targetdir.join(format!("{lib_name}.dll.lib")),
+                        true => targetdir.join(format!("{lib_name}.lib")),
+                    };
```

Comment:
This bit is not entirely correct, under the Meson convention import libs are `lib` but also static libs are changed to `a`.  https://mesonbuild.com/FAQ.html#why-does-building-my-project-with-msvc-output-static-libraries-called-libfooa

Reply:
Static libs behavior is controlled by: 
https://github.com/lu-zero/cargo-c/blob/5a92c3f567a3b9eebb547d8fedcc7de09b042821/src/build_targets.rs#L118-L129

The `shared_lib`, `static_lib` or `pdb` declarations cannot be modified here, as they also define which library is copied from Rust. Consequently, this logic must always align with:
https://github.com/rust-lang/rust/blob/1.84.0/compiler/rustc_target/src/spec/base/windows_msvc.rs#L11-L15